From 95fdac05311da0bb8512fee2044a1575748c497c Mon Sep 17 00:00:00 2001
From: Sven Roederer <devel-sven@geroedel.de>
Date: Sun, 13 Oct 2019 23:12:50 +0200
Subject: [PATCH] mod-network: disable 802.11b legacy-rates by default

change the default to "false" to keep up with the changed default
of hostapd (005-hostapd_nolegacy_by_default.patch)
---
 .../htdocs/luci-static/resources/view/network/wireless.js               | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
index c9f5957f8..0235df0e1 100644
--- a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
+++ b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
@@ -766,7 +766,7 @@ return L.view.extend({
 					o.wifiNetwork = radioNet;
 
 					o = ss.taboption('advanced', form.Flag, 'legacy_rates', _('Allow legacy 802.11b rates'));
-					o.default = o.enabled;
+					o.default = o.disabled;
 
 					o = ss.taboption('advanced', form.Value, 'distance', _('Distance Optimization'), _('Distance to farthest network member in meters.'));
 					o.datatype = 'range(0,114750)';
-- 
2.11.0

